messaging/amqp-reactive/src/test/java/io/quarkus/ts/openshift/messaging/amqp/AbstractAMQPTest.java

```diff
@@ -23,6 +23,6 @@ public void testLastPrice() {
                         .get("/price")
                     .then()
                         .statusCode(200)
-                        .body(equalTo("[10, 20, 30, 40, 50, 60, 70, 80, 90, 100]")));
+                        .body(containsString("10, 20, 30, 40, 50, 60, 70, 80, 90, 100")));
```

Comment:
This solves the issue described in https://github.com/quarkus-qe/quarkus-openshift-test-suite/issues/93#issuecomment-720288017.
Tbh, I didn't really like to change the condition from equals to contains. Though it was the easier solution. Another option would be to change the producer to only produce 10 events...

Reply:
Please add a comment to the AbstractAMQPTest.java saying the same what you have in the above GH comment.

Otherwise good to go!

Reply:
Added. Let me know if it looks good now. Thanks!